### PR TITLE
Wistia Videos Not Showing on Articles

### PR DESCRIPTION
**Pull request by leoalfonso14**
### About
This PR fixes a bug that prevents Wistia videos not showing on articles unless a Video Link is provided as well as adding a Wistia Project media item to the Rock Content Item. 

### Test Instructions
The Testing Wistia Video Content Item on Rock ( [here](https://rock.christfellowship.church/page/2798?ContentItemId=14422) ) has no video linked, however it does have a Wistia Project media item added to it.

1. Go to [/testing-wistia-video](https://www.christfellowship.church/articles/testing-wistia-video) on the website and see how it used to be.
2. Go to [/testing-wistia-video](https://web-app-v2-git-wistia-videos-no-ef3b37-christ-fellowship-church.vercel.app//articles/testing-wistia-video) on the testing link to see how it is with the changes.
3. Remove the Wisitia Project from the Content Item and add just a video link to ensure that it still works with just a link.
4. Test other type of pages that use Content Single like messages or devotionals to make sure nothing else broke.

### Screenshots
|How it used to be|With the changes|
|--|--|
|Screenshot 2023-06-30 at 4 07 33 PM|Screenshot 2023-06-30 at 4 07 33 PM|
|--|--|

### Closes Tickets
[CFDP-2635](https://christfellowshipchurch.atlassian.net/browse/CFDP-2635)

[CFDP-2635]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/ContentSingle/ContentVideo.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ export default function ContentVideo(props = {}) {
         segmentData={props.segmentData}
         wistiaId={props?.wistiaId}
         title={props.title}
-        src={props.video.sources[0].uri}
```
Comment (dzwood) [inline components/ContentSingle/ContentVideo.js:28]: @leoalfonso14 We still want to support regular video URLs as well, so we don't want to get rid of this exactly. We'll have to find a way to support both 🤔

File: components/ContentSingle/ContentVideo.js [hunk 2]
```diff
@@ -36,6 +35,7 @@ export default function ContentVideo(props = {}) {
 
 ContentVideo.propTypes = {
   poster: PropTypes.string,
+  wistiaID: PropTypes.string,
```
Comment (dzwood) [inline components/ContentSingle/ContentVideo.js:38]: Can this be `wistiaId` instead to match prop name?

**Review by dzwood — CHANGES_REQUESTED** [review]

**Comment by dzwood** [comment]
@leoalfonso14 I updated the test item in Rock to just use a Video Link instead of Wistia item. It looks like its not working anymore, so we'll need to find a way to support both Wistia and regular Video URLs. You can see in my requested changes, super great start on this though! 👏

**Review by codingwithlis — APPROVED** [review]
Test passed
Findings:
-If you add a video URL in Rock it will default to the Wistia media unless there is none
-When removing Wistia media in Rock you have to remove `Media` first and then `Wistia Project`, otherwise it doesn't save my changes, this is a Rock issue I assume

**Review by dzwood — APPROVED** [review]
Great job in this! Working great now 🎉

**Pull request merged**